actors/builtin/market/market_state.go

```diff
@@ -3,6 +3,7 @@ package market
 import (
 	addr "github.com/filecoin-project/go-address"
 	"github.com/ipfs/go-cid"
+	xerrors "golang.org/x/xerrors"
```

Comment:
Please use pkg/errors for consistency with the rest of this module.

Reply:
pkg/errors loses a lot of information, and doesnt conform to go's new error handling mechanics. Strongly prefer xerrors (or simply fmt.Errorf, which has the same api, but doesnt preserve stack information)